app/views/roasts/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<%= render "shared/partials/nav-global" %>
+<%= render "shared/partials/header-global" %>
+
+<!-- first loop around main and second loop around roasts_brands section -->
+<!-- can loop the markup through Rails since each roast is templated -->
+
+<section class="roasts__slider-wrapper">
+    <label for="bean-slider" aria-label = "Slide to select a roast!">
+      <div class="slider-axis" aria-hidden = "true">
+        <%= image_tag("light-slider-bean.svg", alt:"") %>
+        <span>Light</span>
+        <span></span>
+        <span></span>
+        <span></span>
+        <span></span>
+        <span></span>
+        <span></span>
+        <span>Dark</span>
+        <%= image_tag("dark-slider-bean.svg", alt:"") %>
+      </div>
+    </label>
+    <div class="slider">
+      <input type="range" id = "bean-slider" value="0" min="0" max="8" step="1" aria-valuemin="0" aria-valuemax="8" aria-valuenow="0" class="slider__range">
+    </div>
```

Comment:
And this input needs a label.

```
<label class="screen-reader-only" for="roast-range">Roast Level: Light to Dark</label>
<input type="range" value="0" min="0" max="8" step="1" class="slider__range" id="roast-range">
```

☝️ note the `for` on the label and the `id` on the input